test/unit/reductions/grid_search/test_grid_search_regression.py

```diff
@@ -154,8 +154,8 @@ def test_bgl_lagrange_specifications():
 
     tradeoffs = [0, 0.25, 0.5, 0.75, 1]
 
-    target1.fit(X, y, aux_data=A, number_of_lagrange_multipliers=len(tradeoffs))
-    target2.fit(X, y, aux_data=A, lagrange_multipliers=tradeoffs)
+    target1.fit(X, y, group_membership=A, number_of_lagrange_multipliers=len(tradeoffs))
+    target2.fit(X, y, group_membership=A, lagrange_multipliers=tradeoffs)
```

Comment:
These extra arguments (Lagrange multipliers) to `fit` shouldn't be there. I must have missed them on my trawl :-(